<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversion</class>
 <widget class="QDialog" name="Conversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversion</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Binary input: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBinIN">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>33</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decimal input: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hexadecimal input: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDecIN">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHexIN">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dec:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dec (from Signed Bin):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBinOUT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDecOUT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1000000000</number>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDecSigOUT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10000000</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHexOUT">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1000000000</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConBin">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConDec">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnConHex">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Signed Bin: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBinSigOUT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
